online-shop/src/app/modules/shared/types/order-create-dto.types.ts

```diff
@@ -0,0 +1,6 @@
+import { ProductIdAndQuantity } from "./product-id-and-quantity.types"
+export interface OrderCreateDto {
```

Comment:
We do not necessary name the types we use in the client app with 'DTO', so you could simply name it 'Order'

Reply:
Renamed.